crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -279,7 +276,7 @@ impl InboundGroupSession {
 
     /// Restore a Session from a previously pickled string.
     ///
-    /// Returns the restored group session or a `OlmGroupSessionError` if there
+    /// Returns the restored group session or a `UnpicklingError` if there
```

Comment:
```suggestion
    /// Returns the restored group session or a `PickleError` if there
```

Probably a left over from the earlier name?